WordPress/src/main/res/layout/actionable_empty_view.xml

```diff
@@ -19,7 +20,7 @@
             <ImageView
                 android:id="@+id/image"
                 android:adjustViewBounds="true"
-                android:contentDescription="@string/content_description_person_reading_device_notification"
+                android:importantForAccessibility="no"
```

Comment:
Do we need to add this to the `bottom_image` `ImageView` as well? Not quite sure what that `ImageView` is for.

Reply:
This is a good question. I am going to investigate.

Reply:
So the `bottom_image` is primarily used in the `GiphyPickerActivity` to show the "Powered by Giphy" logo and there's a content description set on it so when the user hovers they will know that the current page is powered by Giphy. 

https://github.com/wordpress-mobile/WordPress-Android/blob/50f98a16880a535af82c35f7a81c3c65e592fa9a/WordPress/src/main/java/org/wordpress/android/ui/giphy/GiphyPickerActivity.kt#L202

Reply:
Well, now I'm embarrassed because I'm pretty sure I was the one who created that. 😄 I guess it makes sense to keep it. Thanks for taking a look!

Reply:
Haha! Hey! That's how it goes. It's alot of code. Won't remember it all 😄